biorxiv/word_vector_experiment/07_biorxiv_visualize_embeddings_pretrained.py

```diff
@@ -0,0 +1,195 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:annorxiver]
+#     language: python
+#     name: conda-env-annorxiver-py
+# ---
+
+# # Visualize BioRxiv Document Embeddings
+
+# This notebook is designed to visualize umap and tsne representations of bioRxiv document embeddings.
+# Each document embedding is generated via an average of every word in a given article.
+
+# +
+import itertools
+from pathlib import Path
+import re
+
+import pandas as pd
+import plotnine as p9
+# -
+
+journal_map_df = pd.read_csv(
+    "../exploratory_data_analysis/output/biorxiv_article_metadata.tsv", sep="\t"
+)
+journal_map_df.head()
```

Comment:
What do the values in the "heading" column indicate?

Reply:
This "heading" column represents the type of result a preprint is reporting. Majority of preprints report a new result, but some are contradictory or confirmatory research.